view/src/main/resources/static/sass/partials/pages/_userprofile.scss

```diff
@@ -194,6 +194,10 @@ a.attendingConference {
   padding: 20px;
   border: 1px solid #bdbcb7;
   position: relative;
+
+  img {
+    max-width: 100%;
+  }
```

Comment:
Good idea! Since this rule uses an element selector and is not specific to the user profile page, we should move it to a more general file. I think this fits into `base/_layout.scss`.